addons/website/views/snippets/s_numbers_charts.xml

```diff
@@ -0,0 +1,52 @@
+<odoo>
+
+<template id="s_numbers_charts" name="Numbers Charts">
+    <section class="s_numbers_charts pt48 pb48">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="12">
+                <div class="o_grid_item g-col-lg-6 g-height-6" style="grid-area: 1 / 1 / 6 / 7; z-index: 1">
+                    <h3>Key Metrics of Company's<br/>Achievements</h3>
+                    <p class="lead">Our key metrics, from revenue growth to customer retention and market expansion, highlight our strategic prowess and commitment to sustainable business success.</p>
+                </div>
+                <div class="o_grid_item g-col-lg-5 g-height-3" style="grid-area: 6 / 1 / 9 / 6; z-index: 3">
+                    <h4 class="display-4">$ 32M</h4>
+                    <p>Our clients have saved $32 million using our services.</p>
+                    <div class="s_progress_bar s_progress_bar_label_hidden" data-display="inline" data-vcss="001">
+                        <div class="s_progress_bar_wrapper d-flex gap-2">
+                            <div class="progress" role="progressbar" aria-valuenow="80" aria-valuemin="0" aria-valuemax="100">
+                                <div class="progress-bar overflow-visible bg-o-color-2" style="width: 80%; min-width: 3%" aria-label="Progress bar"/>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="o_grid_item g-col-lg-5 g-height-3" style="grid-area: 10 / 1 / 13 / 6; z-index: 2;">
+                    <h4 class="display-4">+25.000</h4>
+                    <p>We proudly serves over 25,000 clients.</p>
+                    <div class="s_progress_bar s_progress_bar_label_hidden" data-display="inline" data-vcss="001">
+                        <div class="s_progress_bar_wrapper d-flex gap-2">
+                            <div class="progress" role="progressbar" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100">
+                                <div class="progress-bar overflow-visible bg-o-color-2" style="width: 45%; min-width: 3%" aria-label="Progress bar"/>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="o_grid_item o_cc o_cc2 g-col-lg-5 g-height-12 rounded" style="grid-area: 1 / 8 / 13 / 13; --grid-item-padding-y: 80px; --grid-item-padding-x: 40px; z-index: 4;">
+                    <div class="s_chart" data-type="doughnut" data-legend-position="none" data-tooltip-display="false" data-stacked="false" data-border-width="2" data-data="{&quot;labels&quot;:[&quot;First&quot;,&quot;Second&quot;],&quot;datasets&quot;:[{&quot;label&quot;:&quot;One&quot;,&quot;data&quot;:[&quot;25&quot;,&quot;75&quot;],&quot;backgroundColor&quot;:[&quot;o-color-3&quot;,&quot;o-color-2&quot;],&quot;borderColor&quot;:[&quot;&quot;,&quot;&quot;]}]}" data-snippet="s_chart" data-name="Chart">
```

Comment:
Ops 😉

```suggestion
                    <div class="s_chart" data-type="doughnut" data-legend-position="none" data-tooltip-display="false" data-stacked="false" data-border-width="2" data-data="{'labels':['First','Second'],'datasets':[{'label':'One','data':['25','75'],'backgroundColor':['o-color-3','o-color-2'],'borderColor':[',']}]}" data-snippet="s_chart" data-name="Chart">
```

Reply:
I kept the entities because I get this error when using real `'`

<img width="1136" alt="Screenshot 2024-08-22 at 09 36 21" src="https://github.com/user-attachments/assets/6d9a3843-8e4a-4d1e-a728-5588ff21ce5d">

Looks like JSON doesn't accept single quotes inside double quotes. So, i could do with the entities or like this:

`data-data='{"labels":["First","Second"],"datasets":[{"label":"One","data":["25","75"],"backgroundColor":["o-color-3","o-color-2"],"borderColor":["",""]}]}'`

I guess website team will enlighten us during TR

Reply:
Yes, `data-data='{"labels":["First","Second"],"datasets":[{"label":"One","data":["25","75"],"backgroundColor":["o-color-3","o-color-2"],"borderColor":["",""]}]}'` it's ok 😉